server/webhook.go

```diff
@@ -296,17 +306,21 @@ func (p *parsedJIRAWebhook) fromChangeLog(issue string) (string, string) {
 			p.event(eventUpdatedResolved)
 			return fmt.Sprintf("resolved %v", issue), ""
 
-		case item.Field == "status" && to == "Backlog":
+		case item.Field == "status" && lto == "in progress":
 			p.event(eventUpdatedStatus)
-			return fmt.Sprintf("moved %v to backlog", issue), ""
+			return fmt.Sprintf("started working on %v", issue), ""
 
-		case item.Field == "status" && to == "In Progress":
+		case item.Field == "status" && lfrom == "in progress" && lto == "to do":
 			p.event(eventUpdatedStatus)
-			return fmt.Sprintf("started working on %v", issue), ""
+			return fmt.Sprintf("stopped working on %v", issue), ""
+
+		case item.Field == "status" && lfrom == "in progress" && lto == "done":
+			p.event(eventUpdatedStatus)
+			return fmt.Sprintf("finished work on %v", issue), ""
 
-		case item.Field == "status" && to == "Selected for Development":
+		case item.Field == "status":
 			p.event(eventUpdatedStatus)
-			return fmt.Sprintf("selected %v for development", issue), ""
+			return fmt.Sprintf("transitioned %v to %q", issue, to), ""
```

Comment:
this is the more generic statement I would expect to see from issue transitions / status updates.

Reply:
Done.